pkg/security/apis/security/validation/validation.go

```diff
@@ -281,9 +281,12 @@ func ValidatePodSecurityPolicySubjectReview(podSecurityPolicySubjectReview *secu
 	return allErrs
 }
 
-func validatePodSecurityPolicySubjectReviewSpec(podSecurityPolicySubjectReviewSpec *securityapi.PodSecurityPolicySubjectReviewSpec, fldPath *field.Path) field.ErrorList {
+func validatePodSecurityPolicySubjectReviewSpec(spec *securityapi.PodSecurityPolicySubjectReviewSpec, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
-	allErrs = append(allErrs, kapivalidation.ValidatePodSpec(&podSecurityPolicySubjectReviewSpec.Template.Spec, fldPath.Child("template", "spec"))...)
+	allErrs = append(allErrs, kapivalidation.ValidatePodSpec(&spec.Template.Spec, fldPath.Child("template", "spec"))...)
+	if len(spec.User) == 0 && len(spec.Groups) == 0 && len(spec.Template.Spec.ServiceAccountName) == 0 {
```

Comment:
I see what you did there :eyes: I wonder whether we should at least add a comment that if none of these are specified, all SCCs would be matched for that PSPSR due to the behavior of `FindApplicableSCCs`, it's not entirely obvious.